docs/src/pages/en/components/media-seek-forward-button.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: <media-seek-forward-button>
+description: Media Seek Forward Button
+layout: ../../../layouts/ComponentLayout.astro
+source: https://github.com/muxinc/media-chrome/tree/main/src/js/media-seek-forward-button.js
+---
+
+import ComponentSandpack from "../../../components/ComponentSandpack";
+
+The `<media-seek-forward-button>` component is used to allow seeking forward a configurable amount of time, defaulting to 30 seconds.
+
+## Default usage
+
+<ComponentSandpack
+  client:load
+  height={290}
+  html={`<media-controller>
+  <video
+    slot="media"
+    src="https://stream.mux.com/A3VXy02VoUinw01pwyomEO3bHnG4P32xzV7u1j1FSzjNg/low.mp4"
+    playsinline
+    muted
+  ></video>
+  <div>
+    <media-play-button></media-play-button
+    ><media-time-display></media-time-display
+    ><media-seek-forward-button></media-seek-forward-button>
+  </div>
```

Comment:
seems less confusing to put these controls already in the `<media-control-bar>` where it's not needed to do the white space tricks.